pinkie-card/GenericTile.qml

```diff
@@ -0,0 +1,56 @@
+import QtQuick 2.7
+import QtGraphicalEffects 1.0
+
+Item {
+    id: generic_tile
+    property int tile_text_spacing_horizontal: 20
+    property int tile_text_spacing_vertical: tile_text_spacing_horizontal
+    property string tile_text: 'Tile'
+    property int tile_text_max_size: 72
+    property color tile_color: 'grey'
+    property int tile_shadow_size: 10
+
+    Text {
+        id: main_text
+        color: "#d9000000"
+        text: qsTr(tile_text)
+        fontSizeMode: Text.Fit
+        clip: true
+        font.weight: Font.Medium
+        font.capitalization: Font.AllUppercase
+        font.pixelSize: tile_text_max_size
+        verticalAlignment: Text.AlignVCenter
+        horizontalAlignment: Text.AlignHCenter
+        anchors.top: background.top
+        anchors.right: background.right
+        anchors.left: background.left
+        anchors.bottom: background.bottom
```

Comment:
These lines may be replaced by "anchors.fill: background". Not absolutely sure though.